spec/system/candidate_interface/candidate_signing_in_spec.rb

```diff
@@ -1,23 +1,54 @@
 require 'rails_helper'
 
 describe 'A candidate signing in' do
+  include TestHelpers::SignUp
+
   before do
     visit '/'
     click_on 'sign in'
 
     fill_in t('authentication.sign_in.email_address.label'), with: 'new_candidate@example.com'
   end
 
-  it 'sees the the sign in page' do
+  it 'sees the sign in page' do
     expect(page).to have_content 'Enter the email address you used to register with'
   end
 
+  context 'when the email does not exist' do
+    it 'sees the "Check your email" page' do
+      fill_in t('authentication.sign_in.email_address.label'), with: 'non_existent_candidate@example.com'
+      click_on t('authentication.sign_up.button')
+
+      expect(page).to have_content 'Check your email'
+      expect(page.current_url).to eq(candidate_interface_sign_in_url)
+    end
+  end
+
   context 'who successfully signs in' do
     it 'sees the "Check your email" page' do
-      fill_in t('authentication.sign_in.email_address.label'), with: 'new_candidate@example.com'
+      fill_in t('authentication.sign_in.email_address.label'), with: 'candidate@example.com'
       click_on t('authentication.sign_up.button')
 
       expect(page).to have_content 'Check your email'
+      expect(page.current_url).to eq(candidate_interface_sign_in_url)
+    end
+
+    it 'receives the email' do
+      visit '/candidate/sign-up'
+      fill_in_sign_up
+
+      visit '/candidate/sign-in'
+      fill_in t('authentication.sign_in.email_address.label'), with: 'april@pawnee.com'
+      click_on 'Continue'
+
+      # This open the last email: sign-in, the first email was sign-up
```

Comment:
Is this comment still relevant?

Reply:
I've put it because of in this flow two different emails are sent, but only the last one is open (in the next row)
But I suppose it is not really important, I'll remove it.